src/test/components/TrackScreenshots.test.js

```diff
@@ -71,8 +71,22 @@ describe('timeline/TrackScreenshots', function() {
   it('moves the hover when moving the mouse', () => {
     const { moveMouseAndGetLeft } = setup();
     const base = moveMouseAndGetLeft(LEFT);
-    expect(moveMouseAndGetLeft(LEFT + 10)).toBe(base + 10);
-    expect(moveMouseAndGetLeft(LEFT + 20)).toBe(base + 20);
+    // LEFT is 100 and the screenshot's width is 300px.
+    // So hovering between 100 and 150px should give a result of left == 0.
```

Comment:
So, in this test, we want to test that the hover moves with the mouse.
So my suggestion is that we compute `base` at a increased value. We know that moving above 150px we'll have a screenshot at the center of the mouse, so we should take the base at 150:
```js
     const base = moveMouseAndGetLeft(150);
```

You can adjust the comment because "150px" isn't the threshold value once we put the RATIO constant back at 1.75. But we can simply say that "150px" is a big enough value to use as a base.